### PR TITLE
Newsletters: add 328 (2024-11-08)

### PR DESCRIPTION
**Pull request by harding**
- [ ] Lede, releases/RCs, topic entries @harding 
- [x] PR Review Club @stickies-v 
- [x] Merge summaries @Gustavojfe

### CODE REVIEW

File: _posts/en/newsletters/2024-11-08-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+This week's newsletter FIXME
+
+## News
+
+- **Disclosure of vulnerabilities affecting Bitcoin Core versions before 25.1:**
+  Antoine Poinsot [announced][poinsot stall] to the Bitcoin-Dev
+  mailing list the final vulnerability disclosure predating Bitcoin
+  Core's new disclosure policy (see [Newsletter #306][news306
+  disclosure]).  The [detailed vulnerability report][stall vuln] notes
+  that Bitcoin Core version 25.0 and earlier were susceptible to an
```
Comment (vostrnad) [inline _posts/en/newsletters/2024-11-08-newsletter.md:19]:
```suggestion
  that Bitcoin Core versions 25.0 and earlier were susceptible to an
```

**Comment by stickies-v** [comment]
Added coverage for yesterday's fresh-of-the-mill review club!

**Comment by harding** [comment]
Made edits for feedback, thanks @vostrnad and @bitschmidty !  Added lede, releases/RCs, and topic entries.  Reviewed PR Club and Merges sections (thanks @stickies-v and @Gustavojfe !).  Thanks everyone!

File: _posts/en/newsletters/2024-11-08-newsletter.md [hunk 2]
```diff
@@ -140,7 +140,7 @@ repo], and [BINANAs][binana repo]._
 [news321 retry]: /en/newsletters/2024/09/20/#ldk-3010
 [review club 30239]: https://bitcoincore.reviews/30239
 [gh instagibbs]: https://github.com/instagibbs
-[28.0 integration guide]: https://bitcoinops.org/en/bitcoin-core-28-wallet-integration-guide/
```
Comment (bitschmidty) [inline _posts/en/newsletters/2024-11-08-newsletter.md]: oops, thanks Dave!

**Review by bitschmidty — COMMENTED** [review]
ACK, lede, review club updates, and topics.

One small (subjective) wording change on the topic.

**Pull request merged**